### ISSUE TITLE
Large int - wrong value returned

### ISSUE THREAD

**Issue by eliascotto** [tiangolo/fastapi#3028]
## Issue

When using a large integer aka postgres `bigint` the value serialization to json is wrong, with a simple request.

### Example

```Python
@router.get("/bigint", status_code=status.HTTP_200_OK)
def get_demo(db: Session = Depends(get_db)):
    value = crud.get_tweet_by_resource_id(db, resource_id='xxxxxxxx').id
    print(value) # 1376946241435750401
    return value

@router.get("/bigint2", status_code=status.HTTP_200_OK)
def get_demo(db: Session = Depends(get_db)):
    return 1377007980063809541
```

#### Ex 1
* Open the browser and call the endpoint `http://127.0.0.1:8000/resources/bigint`.
* It returns a number with `1376946241435750400`.
* But I expected it to return `1376946241435750401`.

#### Ex 2
* Open the browser and call the endpoint `http://127.0.0.1:8000/resources/bigint`.
* It returns a number with `1377007980063809500`.
* But I expected it to return `1377007980063809541`.

### Environment

* OS: macOS 10.15.6 Catalina
* FastAPI Version: 0.62.0
* Python Version: 3.7.4

**Comment by Mause** [comment]
I assume you're calling via the swagger interface? If so, you're running into the limit on numbers in JavaScript

**Comment by elias94** [comment]
>  limit on numbers in JavaScript

So, he really exist 😂

Thanks! Do you know how to fix it other than directly using `str` on the `id` value?

**Comment by Mause** [comment]
You can't really change the number limit in JavaScript/JSON. You can send it as a string then use (BigInt)[https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/BigInt]

**Comment by elias94** [comment]
Yes I was thinking more python side. For the moment I resolved declaring `id: str` on the pydantic models and I does for me the conversion.

**Comment by tiangolo** [comment]
Thanks for the help here @Mause ! 👏  🙇 

Thanks for reporting back and closing the issue @elias94  👍

> Sorry for the long delay! 🙈  I wanted to personally address each issue/PR and they piled up through time, but now I'm checking each one in order.
